### ISSUE TITLE
Determine Project Vue-ness In a Better Way

### ISSUE THREAD

**Issue by nchutchind** [pulilab/vue-people#80]
Currently, the code seems to look at the Languages Github lists for a project to see if it includes "Vue." This is limiting since a project made for Vue may not include any *.vue files, but pure javascript.

Why were tags/topics not used to determine Vue status? It seems like you could look through the topics for a repo for vue, vuejs, etc and get close to the same effect.

**Comment by MaximVanhove** [comment]
Indeed! Or maybe add a check to the filterOutNonVueAndZeroStars if Vue is listed as dependency?

**Comment by lordfuoco** [comment]
@nchutchind @MaximVanhove Agreed that the fetch function needs some improvements / rework.
Let us all came up with the most fair but doable way to do so.
The only constraint is that we must achieve this with only one call to GitHub API ( this should not be a problem since we are using the graphql version )

**Comment by lordfuoco** [comment]
Link to the graphql explorer: https://developer.github.com/v4/explorer/

**Comment by lordfuoco** [comment]
@nchutchind I dug deeper in the the GitHub API and Tags do not seem a good way to go, as far as I understand they are tied to Releases.

About topics: not all the vuejs project have topic set up for example I just adjusted mine to contain those.
But I agree that maybe we should both check: language and topics. Now we would need a list of good topics, or maybe a very very good regex ( I am not too fond of just searching for the word: vue in the topic name)
This is the graphql query that I was playing with: 
```
{
  user(login: "lordfuoco") {
    repositories(first: 100, privacy: PUBLIC) {
      edges {
        node {
          id
          name
          url
          stargazers {
            totalCount
          }
          languages(first: 5, orderBy: {field: SIZE, direction: DESC}) {
            edges {
              node {
                name
              }
            }
          },
          repositoryTopics(first: 10) {
            edges {
              node {
                topic {
                  id,
                  name
                }
              }
            }
          }
        }
      }
    }
  }
}
```

**Comment by lordfuoco** [comment]
I am marking this issue as a good hacktoberfest one, the goal here is to modify the file in `integrations/github` to add to the user repos both repo marked with language: 'vue' and with topics:
https://github.com/topics/vuejs
and 
https://github.com/topics/vue
